context/resolve.go

```diff
@@ -50,6 +54,7 @@ func (ctx *Context) loadPackage() error {
 		if info.IsDir() {
 			return nil
 		}
+		path = strings.Replace(path, rootdir, ctx.RootDir, 1)
```

Comment:
Please also add a comment here that explains why this is necessary and that it is tied to the eval symlink line above.